flaskr.py

```diff
@@ -1,7 +1,39 @@
+from datetime import datetime
 from flask import Flask, render_template, url_for, flash, redirect
+from flask_sqlalchemy import SQLAlchemy
 from forms import RegistrationForm, LoginForm
+
 app = Flask(__name__)
 app.config['SECRET_KEY'] = 'f8db447af15aaeb8af3f4a0d347bef8a' # env. variable
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///site.db'
+db = SQLAlchemy(app)
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(20), unique=True, nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    image_file = db.Column(db.String(20), nullable=False, default='default.jpg')
+    password = db.Column(db.String(60), nullable=False)
+    posts = db.relationship('Post', backref='author', lazy=True)
```

Comment:
**User** Model having _posts_ attribute, which has a relationship to the **Post** Model
backref allows us to get the User who created the Post when we have a Post
lazy argument defines when the SQLAlchemy loads data from DB (True --> load data as necessary in one go)

This is a relationship & NOT A COLUMN